tracker/tracker/src/main/app/index.ts

```diff
@@ -312,8 +312,8 @@ export default class App {
     return this.session.getInfo().sessionID || undefined
   }
 
-  getSessionURL(): string | undefined {
-    const { projectID, sessionID } = this.session.getInfo()
+  getSessionURL(options?: { jumpToCurrentTime?: boolean }): string | undefined {
```

Comment:
could we rename that one to  `withTime` ?

Reply:
@PandaCodes maybe `withCurrentTime` or `withCurrTime`? I want to convey the idea that it's going to include the timestamp of the moment you called the function.

Reply:
`withCurrentTime`  should be also ok